airflow/serialization/serialized_objects.py

```diff
@@ -1235,6 +1235,7 @@ def _serialize_node(cls, op: BaseOperator | MappedOperator) -> dict[str, Any]:
 
         # Used to determine if an Operator is inherited from EmptyOperator
         serialize_op["_is_empty"] = op.inherits_from_empty_operator
+        serialize_op["_is_skip_mixin"] = op.inherits_from_skip_mixin
```

Comment:
I wonder if we could call this `_is_branching` or `_is_conditional` or something -- thinking ahead a bit if we have DAGs defined in other languages (be it YAML or anything else) I'm not sure naming it as skip_mixin makes sense anymore.

Reply:
Looking at how it's used, `if parent.inherits_from_skip_mixin`, I think `if parent.is_conditional` would work there. That `is_conditional` name _might_ be a little too generic though.

Reply:
Your argument makes sense, let's settle for `is_skippable`?
`is_conditional` sounds indeed too generic.